test/manual/maps/map_test.R

```diff
@@ -0,0 +1,193 @@
+  return(p)
+}
+
+
+#' Plot randomly generated points over basemap created from in map_points.R
+#'
+#' @param iso3 ISO3 code
+#' @param sample_n `integer` vector (default = 1:20) sampled to randomly decide how many
+#'    pts will be mapped
+#' @param sample_values `integer` vector (defaults 1:20000) sampled to assign values to pts.
+#'    might be useful to have it sample a log distribution instead
+#' @param use_bbox `logical` if TRUE (default) samples will be drawn from the bounding box
+#'    of the polygon, if FALSE sampled withing the polygon. It's mainly a feature to
+#'    ease development as sampling withing bbox is much faster
+#' @param subtitle `character` subtitle to label plot
+#'
+#' @return plot ggplot
+
+map_with_points_test  <- function(
+  iso3,
+  sample_n = 1:20,
+  sample_values = 1:20000,
+  use_bbox = TRUE
+) {
+
+  gdf_sample_pts <- sample_pts_iso3(
+    iso3 = iso3,
+    sample_n = sample_n,
+    sample_values = sample_values,
+    use_bbox = use_bbox
+  )
+
+  map_points$map_points(
+    iso3 = iso3,
+    df = gdf_sample_pts,
+    val_col = "value",
+    size = "Test bubble",
+    subtitle = iso3
+  )
+}
+
+#' Generate pts for test map from iso3 code
+#'
+#' @param iso3 ISO3 code
+#' @param sample_n `integer` vector (default = 1:20) sampled to randomly decide how many
+#'    pts will be mapped
+#' @param sample_values `integer` vector (defaults 1:20000) sampled to assign values to pts.
+#'    might be useful to have it sample a log distribution instead
+#' @param use_bbox `logical` if TRUE (default) samples will be drawn from the bounding box
+#'    of the polygon, if FALSE sampled withing the polygon. It's mainly a feature to
+#'    ease development as sampling withing bbox is much faster
+#'
+#' @return sf class data.frame with point geometry
+
+sample_pts_iso3 <- function(
+    iso3,
+    sample_n = 1:20,
+    sample_values = 1:20000,
+    use_bbox = TRUE) {
+  gdf_adm0 <- get_iso3_sf$get_iso3_sf(iso3, "adm0")
+
+  random_spatial_sample(
+    poly = gdf_adm0,
+    use_bbox = use_bbox,
+    sample_n = sample_n,
+    sample_values = sample_values
+  )
+}
+
+#' Randomly generate points within polygon
+#'
+#' Generates a point sample within a sample with a flexible number of points and point values.
+#' This is used withing the map_test() function to simulate points for visual inspection.
+#'
+#' @param poly sf class polygon
+#' @param sample_n `integer` vector (default = 1:20) sampled to randomly decide how many
+#'    pts will be mapped
+#' @param sample_values `integer` vector (defaults 1:20000) sampled to assign values to pts.
+#'    might be useful to have it sample a log distribution instead
+#' @param use_bbox `logical` if TRUE (default) samples will be drawn from the bounding box
+#'    of the polygon, if FALSE sampled withing the polygon. It's mainly a feature to
+#'    ease development as sampling withing bbox is much faster
+#'
+#' @return sf class data.frame with point geometry
+#'
+#' @examples \dontrun{
```

Comment:
You shouldn't put `dontrun` in your examples. `dontrun` means users can't automatically run them (which we definitely want to be able to do) AND that it is not run in testing. `donttest` means users can run them, but still won't run in tests. You should only use either if there is something in the example like, optional packages or API key requirements, but none apply here.